roles/mailserver/tasks/postfix.yml

```diff
@@ -16,12 +16,14 @@
   tags:
     - dependencies
 
-- name: Set postgres password
-  command: sudo -u {{ db_admin_username }} psql -d {{ db_admin_username }} -c "ALTER USER postgres with  password '{{ db_admin_password }}';"
+- name: Set postgres user password
```

Comment:
the comment is misleading, it sets the admin password. How about:
"Set password for ProgreSQL admin user"